Backpack-SwiftUI/AppSearchModal/Classes/BPKAppSearchModal.swift

```diff
@@ -86,12 +86,13 @@ public struct BPKAppSearchModal: View {
                     .accessibilityHidden(false)
                     .accessibilityRemoveTraits(.isImage)
                     .accessibilityAddTraits(.isButton)
-                    .accessibilityLabel(closeAccessibilityLabel)
+                    .accessibilityLabel(closeAccessibilityLabel.lowercased())
```

Comment:
we pretty sure need to remove that
I did it because UITests didn't see any element with non-lowercased localisable string in it
the issue was 100% reproducible...until I restart my machine (which I did just in case)
so, now I don't see any problems with that